backend-tests/tests/test_devauth.py

```diff
@@ -833,6 +851,9 @@ def test_device_count(self, devs_authsets, user):
 
 
 class TestDeviceMgmtEnterprise(TestDeviceMgmtBase):
+    @pytest.mark.skipif(
+        isK8S(), reason="not feasible to test with existing tenant",
+    )
```

Comment:
I'm not 100% sure it makes sense to skip thesee tests if `isK8S()` is True.
We are running these tests in staging today, without reusing existing tenants.
Maybe we should mark it differently, based on an environment variable we'll set when running in "existing tenant" mode?

Reply:
Using different variable make sense; I'll update the PR.

Reply:
done